tests/testthat/test-stacks.R

```diff
@@ -129,25 +129,25 @@ test_that("integration tests", {
   df <- causeError(full = FALSE)
   # dumpTests(df)
 
-  expect_equal(df$num, c(56L, 55L, 54L, 38L, 37L, 36L, 35L,
+  expect_equal(df$num, c(57L, 56L, 55L, 54L, 38L, 37L, 36L, 35L,
     34L, 33L, 32L, 31L, 30L))
-  expect_equal(df$call, c("A", "B", "<reactive:C>", "C", "renderTable",
+  expect_equal(df$call, c("A", "B", "reactive", "<reactive:C>", "C", "renderTable",
     "func", "force", "withVisible", "withCallingHandlers", "domain$wrapSync",
     "promises::with_promise_domain", "captureStackTraces"))
-  expect_equal(nzchar(df$loc), c(TRUE, TRUE, TRUE, FALSE, TRUE,
+  expect_equal(nzchar(df$loc), c(TRUE, TRUE, TRUE, FALSE, FALSE, TRUE,
     FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE))
 
   df <- causeError(full = TRUE)
   # dumpTests(df)
 
-  expect_equal(df$num, c(59L, 58L, 57L, 56L, 55L, 54L, 53L,
+  expect_equal(df$num, c(60L, 59L, 58L, 57L, 56L, 55L, 54L, 53L,
     52L, 51L, 50L, 49L, 48L, 47L, 46L, 45L, 44L, 43L, 42L, 41L,
     40L, 39L, 38L, 37L, 36L, 35L, 34L, 33L, 32L, 31L, 30L, 29L,
     28L, 27L, 26L, 25L, 24L, 23L, 22L, 21L, 20L, 19L, 18L, 17L,
     16L, 15L, 14L, 13L, 12L, 11L, 10L, 9L, 8L, 7L, 6L, 5L, 4L,
     3L, 2L, 1L))
   expect_equal(df$call, c("h", ".handleSimpleError", "stop",
-    "A", "B", "<reactive:C>", "..stacktraceon..", ".func", "withVisible",
+    "A", "B", "reactive", "<reactive:C>", "..stacktraceon..", ".func", "withVisible",
     "withCallingHandlers", "contextFunc", "env$runWith", "force",
     "domain$wrapSync", "promises::with_promise_domain",
     "withReactiveDomain", "domain$wrapSync", "promises::with_promise_domain",
```

Comment:
@jcheng5 These changes are necessary due to calling `installExprFunction()` which adds a function label. (Where before was `exprToFunction()`, which didn't wrap the function.)

Are these changes ok? I saw many messages warning about interactions with promises, but I believe the changes are expected changes.

Reply:
Changes reverted. Closing